packages/slider/src/Slider.ts

```diff
@@ -143,51 +165,39 @@ export class Slider extends Focusable {
         `;
     }
 
-    protected updated(changedProperties: PropertyValues): void {
-        if (changedProperties.has('value')) {
-            this.dispatchInputEvent();
-        }
+    public connectedCallback(): void {
+        super.connectedCallback();
+        this.handleController.hostConnected();
+        this.addEventListener(
```

Comment:
I feel like this is an implementation for more flexible reuse, which is cool, but we might want to just allow the controller too access `this.host.requestUpdate()` to simplify this for our use case?

Reply:
Are you talking about the event listener? It is listening to the slider handle, which the user owns in case they set the value programatically.

Reply:
Ah, ok,  that makes sense.